data/data-pipeline/data_pipeline/tests/score/fixtures.py

```diff
@@ -2,6 +2,9 @@
 import pytest
 from data_pipeline.config import settings
 from data_pipeline.score import field_names
+from data_pipeline.etl.score import constants
+
+GEOID_TRACT_FIELD_NAME = field_names.GEOID_TRACT_FIELD
```

Comment:
an extremely annoying nit: we set this constant here and then lower, we use `field_names.GEOID_TRACT_FIELD` anyway

Reply:
thank you.